test/groovy/MTABuildTest.groovy

```diff
@@ -132,9 +110,9 @@ public class MtaBuildTest extends BasePiperTest {
     void badMtaTest() {
 
         thrown.expect(ParserException)
-        thrown.expectMessage('while parsing a block mapping')
+        //thrown.expectMessage('while parsing a block mapping')
```

Comment:
I think we should delete that line rather then comment

Reply:
fixed, was unintentionally